### PR TITLE
Show dishes

### PR DESCRIPTION
**Pull request by phucld95**

### CODE REVIEW

File: restaurant_management/app/controllers/tables_controller.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
 class TablesController < ApplicationController
   def index
-    if params[:table]
-      from = params[:table][:day] + " " + params[:table][:from]
-      to = params[:table][:day] + " " + params[:table][:to]
-      @tables = Table.free params[:table][:capacity], from, to
+  	if params[:commit] == "Go book table"
+        redirect_to new_guest_path
+    else
+			if params[:table]
```
Comment (dat-hedspi) [inline restaurant_management/app/controllers/tables_controller.rb]: indent 1 đống

File: restaurant_management/app/views/order_tables/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        placeholder: (t "order_table.index.from") %>
+    </div>
+    <div class="col-md-4 col-sm-4">
+      <%= f.time_field :book_to, class: "form-control",
+        placeholder: (t "order_table.index.to") %>
+    </div>
+    <div class="col-sm-6">
+      <%= f.submit t "order_table.index.submit" %>
+    </div>
+     <div class="col-sm-6">
```
Comment (dat-hedspi) [inline restaurant_management/app/views/order_tables/_form.html.erb]: indent

File: restaurant_management/app/views/order_tables/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        <h1 class="heading">BOOK TABLE</h1>
+        <hr>
+      </div>
+      <div class="col-md-offset-1 col-md-10 col-sm-12">
+        <%= render "form" %>
+      </div>
+      <div class="col-md-2 col-sm-1"></div>
+    </div>
+  </div>
+</section>
```
Comment (dat-hedspi) [inline restaurant_management/app/views/order_tables/new.html.erb]: end file

File: restaurant_management/app/controllers/guests_controller.rb [hunk 4]
```diff
@@ -2,8 +2,7 @@ class GuestsController < ApplicationController
   def create
     @guest = Guest.new guest_params
     if @guest.save
-      flash[:success] = t "flash.step3"
-      redirect_to @user
+      redirect_to new_guest_order_table_path(@guest)
```
Comment (yeunuthongminh) [inline restaurant_management/app/controllers/guests_controller.rb]: remove ()

**Comment by dat-hedspi** [comment]
OK

File: restaurant_management/app/controllers/guests_controller.rb (hunk 4 above)
Comment (yeunuthongminh) [inline restaurant_management/app/controllers/guests_controller.rb]: remove ()

File: restaurant_management/app/views/order_tables/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<%= form_for [@guest, @order_table], url:  guest_order_tables_path, method: :post do |f| %>
+  <div class="col-sm-12 dropdown-buttons">
+    <div class="col-md-12 col-sm-12">
+      <%= f.number_field :table_id, min: 1, max: 20, class: "form-control", 
+        placeholder: ("Table Number") %>
```
Comment (yeunuthongminh) [inline restaurant_management/app/views/order_tables/_form.html.erb]: i18n

File: restaurant_management/app/views/order_tables/_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,34 @@
+<%= form_for [@guest, @order_table], url:  guest_order_tables_path, method: :post do |f| %>
+  <div class="col-sm-12 dropdown-buttons">
+    <div class="col-md-12 col-sm-12">
+      <%= f.number_field :table_id, min: 1, max: 20, class: "form-control",
```
Comment (yeunuthongminh) [inline restaurant_management/app/views/order_tables/_form.html.erb]: setings
khai báo trong min max trong model

File: restaurant_management/app/views/order_tables/_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,34 @@
+<%= form_for [@guest, @order_table], url:  guest_order_tables_path, method: :post do |f| %>
```
Comment (yeunuthongminh) [inline restaurant_management/app/views/order_tables/_form.html.erb]: url: guest_order_tables_path

**Comment by yeunuthongminh** [comment]
OK

**Pull request merged**